src/fontfamily/fontfamilyui.js

```diff
@@ -0,0 +1,117 @@
+/**
+ * @license Copyright (c) 2003-2018, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+/**
+ * @module font/fontfamily/fontfamilyui
+ */
+
+import Plugin from '@ckeditor/ckeditor5-core/src/plugin';
+import Model from '@ckeditor/ckeditor5-ui/src/model';
+import Collection from '@ckeditor/ckeditor5-utils/src/collection';
+
+import { createDropdown, addListToDropdown } from '@ckeditor/ckeditor5-ui/src/dropdown/utils';
+
+import { normalizeOptions } from './utils';
+
+import fontFamilyIcon from '../../theme/icons/font-family.svg';
+
+/**
+ * @extends module:core/plugin~Plugin
```

Comment:
What does it do? What sort of UI does it create? How to use it in the configuration?